setup.cfg

```diff
@@ -11,3 +11,4 @@ filterwarnings =
     error
     ignore:Legacy index templates are deprecated in favor of composable templates.:elasticsearch.exceptions.ElasticsearchWarning
     ignore:datetime.datetime.utcfromtimestamp\(\) is deprecated and scheduled for removal in a future version..*:DeprecationWarning
+asyncio_mode = auto
```

Comment:
Please keep the default (strict mode) as this is more explicit and will help to add Trio support in the future.

Reply:
I'm not sure how would that work with unasync. This is one example in which the code would need to be different. Async tests need a decorator to declare them as async, but that decorator needs to be removed in the sync conversion.

Example:

_async version:

```python
@pytest.mark.asyncio
async def test_some_asyncio_code():
    res = await library.do_something()
    assert b"expected result" == res
```

_sync version:

```python
def test_some_asyncio_code():
    res = library.do_something()
    assert b"expected result" == res
```

How do I tell unasync to remove the decorator here?